packages/core/user-profile/core-user-profile-browser-internal/src/utils/default_implementation.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { of } from 'rxjs';
+import type {
+  CoreUserProfileDelegateContract,
+  GetUserProfileResponse,
+} from '@kbn/core-user-profile-browser';
+import { UserProfileData } from '@kbn/core-user-profile-common';
+
+export const getDefaultUserProfileImplementation = (): CoreUserProfileDelegateContract => {
+  return {
+    userProfile$: of(null),
+    getCurrent: <D extends UserProfileData>() =>
```

Comment:
Returning `null` here even if not strictly allowed, for simplicity's sake (to avoid forging a proper userProfile data with the right shape) . It's fine for the default implementation given it's only present for test env (IT tests)